src/flash.rs

```diff
@@ -48,21 +47,22 @@ pub enum FlashSize {
     Sz1M = 1024,
 }
 impl FlashSize {
-    const fn kbytes(self) -> u32 {
+    const fn bytes(self) -> u32 {
```

Comment:
Changed tthis to bytes since, from what I can tell really returns a value in units of bytes